tests/test/storage/test_shared_files.cpp

```diff
@@ -1,32 +1,47 @@
 #include <catch2/catch.hpp>
 
+#include "faabric/util/func.h"
+#include "faasm_fixtures.h"
 #include "utils.h"
 
 #include <boost/filesystem.hpp>
 
 #include <conf/FaasmConfig.h>
-#include <conf/function_utils.h>
 #include <faabric/util/files.h>
+#include <storage/FileLoader.h>
 #include <storage/SharedFiles.h>
 
 using namespace storage;
 
 namespace tests {
-TEST_CASE("Check sync shared file", "[storage]")
+
+class SharedFilesTestFixture : public S3TestFixture
```

Comment:
Why isn't this under `utils/faasm_fixtures.h`? What is the criterion to include fixtures or not?

I assume we include fixtures there if they are used in more than one file, that works for me, but just double-checking.

Reply:
Yes it's not a hard and fast rule but generally fixtures should contain things that are useful to testing more than one _component_ of the system (this is _mostly_ indicated by being needed in more than one test file). In this case, this logic is only needed for testing the shared files, so doesn't merit a fixture. However, the other fixture in this PR where it reads in wasm and object files up front, is useful for testing machine code generation and shared file handling, so is in a fixture. 

All that said, we do have to be careful to keep the fixtures simple, and make sure we're not testing the same system component in multiple files (as this suggests wasted effort or a poor class/ module structure). Again though, no hard and fast rules, just make a call case-by-case.